src/components/PageLevelBanner/__snapshots__/PageLevelBanner.test.tsx.snap

```diff
@@ -25,24 +25,11 @@ exports[`<PageLeveBanner /> Brand story renders snapshot 1`] = `
     >
       New curriculum updates are available for one or more of your courses.
     </h3>
-    <p
-      class="text text--sm text--neutral"
-    >
-      Summit Learning has a full-time team dedicated to constantly improving our curriculum. To see the updates,
-       
-      <button
-        class="clickable-style button button--link clickable-style--link clickable-style--brand"
-        type="button"
-      >
-        click into the course
-      </button>
-      .
-    </p>
```

Comment:
Looks like the change in `.stories` is messing up the `description` prop value generation logic 🤔 cc @jinlee93

Reply:
Actually we don't need that `getDescription` method anymore since the link will always has the "neutral" status.